includes/core/class-member-directory.php

```diff
@@ -1710,7 +1710,8 @@ protected function prepare_search( $search ) {
 			$regexp_map = array(
 				'/select(.*?)from/im',
 				'/select(.*?)sleep/im',
-				"/sleep\(\d+\)/im", // avoid any sleep injections
+				"/sleep\(\s*[\"'`]?\s*\d+\s*[\"'`]?\s*\)/im", // avoid any sleep injections
```

Comment:
What do you think about

```suggestion
				"/sleep\([^)]+)/im", // avoid any sleep injections
```

Reply:
Thanks @sjinks, it looks easier and works. It will also catch no-standard syntax inside sleep.